Xamarin.Forms.Platform.GTK/Renderers/ScrollViewRenderer.cs

```diff
@@ -34,8 +34,8 @@ protected override void OnElementChanged(ElementChangedEventArgs<ScrollView> e)
                         CanFocus = true,
                         ShadowType = ShadowType.None,
                         BorderWidth = 0,
-                        HscrollbarPolicy = PolicyType.Automatic,
-                        VscrollbarPolicy = PolicyType.Automatic
+                        HscrollbarPolicy = ScrollBarVisibilityToGtk(Element.HorizontalScrollBarVisibility),
```

Comment:
As far as I can tell the GTK ScrolledWindow doesn't honer the Scrollbar policy.  Or some other action is required to get it to refresh after setting the value.  Someone who knows more about GTK might have to look at this.